step_by_step/README.md

```diff
@@ -2,59 +2,95 @@
+- Review the [Organizer FAQ](https://faq-organizers.djangogirls.org/) to help optimize the organization process and get you prepared.
+- Review the [accessibility information](.//accessibility/README.md). This includes Website compliance, venue requirements, and other resources you might need to accommodate accessibility needs of your attendees.
+- __Apply to the Django Girls organization to register your event__ by [filling out this form](https://djangogirls.org/organize/).
+- [Adapt the Website template](../website/README.md) that you receive from the support team for your event and launch it. The Website template includes placeholders for information about the venue, application process, coaches, sponsors, and FAQ for applicants.
+- Review and download the resources you might want to use. We have some cool [design resources](../resources/README.md), like Web and print logos, shirt designs, poster templates, and other goodies that you can use in your decorations.
+- [Create a budget for the workshop](../costs/README.md). The budget should be divided to "must-have" costs and "nice-to-have" extra items. Don't worry about raising a lot of money, normally you do not need a big budget, as long as your team of coaches and supporters provide a friendly and fun atmosphere. You can also [DIY a lot of the logistics](../diy/README.md) if you don't feel comfortable handling large amounts of money or depend on sponsorships. Get creative!
+- Start promoting the workshop [on social media, in the press, or in person](../promotion/README.md). You can create a social media account, group, or page.
+- [Start looking for sponsors](../sponsors/README.md). Typical sponsors can include IT companies, universities, and local businesses in your city. You don't need a lot of starting funds to run a workshop, but having financial support can help you provide financial aid for applicants who are not able to travel to your workshop, as well as purchase goodies like shirts for coaches, decorations, and other extra-special treats for a fun day!
+- [Start looking for a venue](../logistics/venue.md). If you are organizing a local workshop, you might reach out to a local IT company or educational facility, who normally are happy to host our workshops. You can also [co-locate the workshop with a conference](../conferences/README.md), which can help share the logistics work with the conference organizers and give the participants an opportunity to interact with a relevant open-source community.
+- [Start looking for coaches](../coaches/README.md). Coaches can be local Python or Django folks, graduates of previous Django Girls workshops, orgaizers, and so on. You can reach out to local Python or Django user groups or meetups, post a request on our [Slack](https://djangogirls.slack.com/?) in the #organizers channel.
```

Comment:
Typo: orgaizers -> organizers.

The last sentence also doesn't read quite right, maybe wants another "or" ("or post a request")